openassessment/xblock/static/sass/oa/views/_oa-base.scss

```diff
@@ -500,6 +500,10 @@
     }
   }
 
+  .submission__answer__part__text__title {
+    @extend %t-subheading;
```

Comment:
This is what's causing the bold labels - is that acceptable? I think it looks better than before.